DataScience/IntroToPython/IntroToPythonPandas/blog.md

```diff
@@ -82,15 +82,13 @@ We have our data with us, but it’s very large, so we might be confused as to w
 |:-----:|------:|:-----:|------:|
```

Comment:
The introduction looks good- maybe specify "low learning curve" (is this describing how Python is easy to use?). I am also curious about the "Python is heavy on English-level syntax" phrase, since the intro is describing how easy Python is easy to use.